### PR TITLE
Rename corpus to index

### PR DESCRIPTION
**Pull request by pwoznic**
Renamed corpus to index in 4 topics

### CODE REVIEW

File: www/docs/api-reference/admin-apis/delete-corpus.md [hunk 1]
```diff
@@ -8,20 +8,20 @@ import {Config} from '@site/docs/definitions.md';
 import {vars} from '@site/static/variables.json';
 
 <Config v="names.product"/> exposes a REST endpoint at the following URL
-to index content into a corpus:
+to injest content into an index:
```
Comment (eskibars) [inline www/docs/api-reference/admin-apis/delete-corpus.md:11]: s/injest/ingest/

File: www/docs/api-reference/admin-apis/delete-corpus.md [hunk 2]
```diff
@@ -8,20 +8,20 @@ import {Config} from '@site/docs/definitions.md';
...
 <code>https://<Config v="domains.rest.admin"/>/v1/delete-corpus</code>
 This page describes the details of interacting with this endpoint.
 
-To delete a corpus, specify the **customer id** and **corpus id**. Upon
-successful completion, space quota consumed by the corpus will be freed,
-and the corpus will no longer be usable for future indexing or querying.
-Note that the corpus id assigned to the corpus will not be reused.
+To delete an index, specify the **customer id** and **corpus id**. Upon
+successful completion, space quota consumed by the index will be freed,
+and the index will no longer be usable for future injesting or querying.
```
Comment (eskibars) [inline www/docs/api-reference/admin-apis/delete-corpus.md:17]: s/injesting/ingesting/

File: www/docs/api-reference/admin-apis/create-corpus.md [hunk 3]
```diff
@@ -10,12 +10,12 @@ import {vars} from '@site/static/variables.json';
 ## Endpoint Address
 
 <Config v="names.product"/> exposes a REST endpoint at the following URL
-to index content into a corpus:
+to injest content into an index:
```
Comment (eskibars) [inline www/docs/api-reference/admin-apis/create-corpus.md:13]: s/injest/ingest/

**Review by eskibars — CHANGES_REQUESTED** [review]
A few spelling modifications.  I'm OK with this, but I want to be prepared to revert if this doesn't meaningfully improve corpus creation metrics

File: www/docs/api-reference/admin-apis/delete-corpus.md (hunk 1 above)
Comment (pwoznic) [reply www/docs/api-reference/admin-apis/delete-corpus.md]: Whoops 😬

**Comment by pwoznic** [comment]
I fixed the typos in the latest commit

**Comment by danielone** [comment]
@pwoznic Great work! 
Only addition is I think we should introduce index/indicies verbiage in the body of creating a corpus
https://deploy-preview-77--luxury-shortbread-acee05.netlify.app/docs/console-ui/creating-a-corpus

Noteably the first line of text
"To get begin searching your data, you first have to create a corpus. A corpus is a container where you upload all your data to be indexed and grouped together in a single location for querying."

You may want to consider lightly adjusting some of the follow on text regarding corpus vs. index when it comes to what users should be clicking on and removing images as well, perhaps just including a single one. 

This document is directly linked to from the console. This way the flow from console -> docs for create corpus can be tied together in a minimal way.

**Comment by pwoznic** [comment]
> @pwoznic Great work! Only addition is I think we should introduce index/indicies verbiage in the body of creating a corpus https://deploy-preview-77--luxury-shortbread-acee05.netlify.app/docs/console-ui/creating-a-corpus
> 
> Noteably the first line of text "To get begin searching your data, you first have to create a corpus. A corpus is a container where you upload all your data to be indexed and grouped together in a single location for querying."
> 
> You may want to consider lightly adjusting some of the follow on text regarding corpus vs. index when it comes to what users should be clicking on and removing images as well, perhaps just including a single one.
> 
> This document is directly linked to from the console. This way the flow from console -> docs for create corpus can be tied together in a minimal way.

Done :)

File: www/docs/api-reference/admin-apis/create-corpus.md [hunk 4]
```diff
@@ -33,21 +33,21 @@ message CreateCorpusRequest {
...
 message CreateCorpusResponse {
-  // The Corpus ID that was created.
+  // The index is the Corpus ID that was created.
   uint32 corpus_id = 1;
   Status status = 2;
 }
 
 message Corpus {
-  // The Corpus ID.
+  // The index ID.
```
Comment (cjcenizal) [inline www/docs/api-reference/admin-apis/create-corpus.md:42]: I can imagine folks being confused by this -- "Index ID? Corpus ID? What's the difference?" Can we clarify things a bit here? For example: "The index ID, also referred to as the corpus ID."

File: www/docs/api-reference/admin-apis/delete-corpus.md [hunk 5]
```diff
@@ -8,20 +8,20 @@ import {Config} from '@site/docs/definitions.md';
...
 <code>https://<Config v="domains.rest.admin"/>/v1/delete-corpus</code>
 This page describes the details of interacting with this endpoint.
 
-To delete a corpus, specify the **customer id** and **corpus id**. Upon
-successful completion, space quota consumed by the corpus will be freed,
-and the corpus will no longer be usable for future indexing or querying.
-Note that the corpus id assigned to the corpus will not be reused.
+To delete an index, specify the **customer id** and **corpus id**. Upon
+successful completion, space quota consumed by the index will be freed,
+and the index will no longer be usable for future ingesting or querying.
```
Comment (cjcenizal) [inline www/docs/api-reference/admin-apis/delete-corpus.md:17]: In this sentence, "querying" refers to the index as in "the index is queried". But "ingesting" doesn't refer to index in the same way -- "the index is ingested" is incorrect. I think it might be clearer to rephrase this as:

> the index will no longer be usable for data ingestion or querying

File: www/docs/api-reference/admin-apis/reset-corpus.md [hunk 6]
```diff
@@ -8,17 +8,17 @@ import {Config} from '@site/docs/definitions.md';
...
 
 <Config v="names.product"/> exposes a REST endpoint at the following URL
-to index content into a corpus:
+to reset an index:
 <code>https://<Config v="domains.rest.admin"/>/v1/reset-corpus</code>
 This page describes the details of interacting with this endpoint.
 
-To reset a corpus, specify the **customer id** and **corpus id**. Upon
-successful completion, space quota consumed by the corpus will be freed.
+To reset an index, specify the **customer id** and **corpus id**. Upon
```
Comment (cjcenizal) [inline www/docs/api-reference/admin-apis/reset-corpus.md:15]: Nit: Should "customer id" be `customer_id`, and "corpus id" be `corpus_id`, since we're referring to parts of the interface?

File: www/docs/api-reference/admin-apis/delete-corpus.md [hunk 7]
```diff
@@ -8,20 +8,20 @@ import {Config} from '@site/docs/definitions.md';
...
+Note that the corpus_id assigned to the index will not be reused.
 
 ```protobuf
 message DeleteCorpusRequest {
   // The Customer ID that contains the corpus to be deleted.
   uint32 customer_id = 1;
-  // The Corpus ID to be deleted.
+  // The index is the Corpus ID to be deleted.
```
Comment (cjcenizal) [inline www/docs/api-reference/admin-apis/delete-corpus.md:24]: Similar concern here as above. Maybe, "The corpus ID uniquely identifies the index to delete"?

File: www/docs/api-reference/admin-apis/reset-corpus.md [hunk 8]
```diff
@@ -8,17 +8,17 @@ import {Config} from '@site/docs/definitions.md';
...
+successful completion, space quota consumed by the index will be freed.
 
 ```protobuf
 message ResetCorpusRequest {
   uint32 customer_id = 1;
-  // The Corpus ID to be reset.
+  // The index is the Corpus ID to be reset.
```
Comment (cjcenizal) [inline www/docs/api-reference/admin-apis/reset-corpus.md:21]: Similar concern here as above. Maybe, "The corpus ID uniquely identifies the index to reset"?

**Review by cjcenizal — APPROVED** [review]
Changes look great! Just had a few suggestions.

File: www/docs/api-reference/admin-apis/reset-corpus.md (hunk 8 above)
Comment (pwoznic) [reply www/docs/api-reference/admin-apis/reset-corpus.md]: I made the suggested changes for all of these

**Comment by pwoznic** [comment]
Updated topics for more clarifications

**Review by danielone — APPROVED** [review]
